cpp/src/slice2py/PythonUtil.cpp

```diff
@@ -425,7 +425,7 @@ Slice::Python::createCodeFragmentForPythonModule(const ContainedPtr& contained,
     fragment.sliceFileName = contained->file();
     string fileName = fragment.moduleName;
     replace(fileName.begin(), fileName.end(), '.', '/');
-    fragment.fileName = fileName + (isForwardDeclaration ? "_iceF.py" : ".py");
```

Comment:
we were adding `_iceF.py` twice , I plan to replace with `_forward.py` in a follow up PR.